jobs/config.json

```diff
@@ -11578,7 +11578,8 @@
     "args": [
       "make",
       "verify",
-      "WHAT=typecheck"
+      "WHAT=typecheck",
+      "KUBE_JUNIT_REPORT_DIR=/workspace/artifacts"
```

Comment:
use $WORKSPACE, we expand env https://github.com/kubernetes/test-infra/blob/master/scenarios/execute.py

Reply:
`KUBE_JUNIT_REPORT_DIR=${WORSPACE}/artifacts`, I think

Reply:
Nothing in execute.py expands environment variables.

Reply:
I think it does:

```
prow$ ../scenarios/execute.py make get-cluster-credentials FOO=$HOME
Run: ('make', 'get-cluster-credentials', 'FOO=/usr/local/google/home/senlu')
```

my concern is $WORKSPACE might be `/go/src` instead of `/workspace`? (in podspec we set `--root=/go/src`)

Reply:
er sorry, bootstrap expands env

Reply:
not the execute scenario

Reply:
Done.

@krzyzacy that was your shell expanding $HOME, not execute.py :-) ('FOO=$HOME' wouldn't get expanded)

Reply:
ah.. :100: